packages/experimental/RadioGroup/src/RadioGroup/RadioGroup.tsx

```diff
@@ -55,7 +56,7 @@ export const RadioGroup = compose<RadioGroupType>({
           <Slots.root {...mergedProps}>
             {label && labelComponent}
             <Slots.container isCircularNavigation defaultTabbableElement={defaultTabbableElement}>
-              {children}
+              <Slots.options>{children}</Slots.options>
```

Comment:
It seems like we're adding an extra View just for the layout stuff - is it possible for the layout stuff in Slots.options to be included in Slots.container? Not really familiar with FocusZone though so not sure if that's appropriate/possible to do or not